src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -58,6 +58,7 @@ public static GetTranscriptionByIdResponse mapToTranscriptionResponse(Transcript
             CourtCaseEntity courtCase = transcriptionEntity.getCourtCase();
             transcriptionResponse.setTranscriptionId(transcriptionEntity.getId());
             transcriptionResponse.setCaseId(courtCase.getId());
+            transcriptionResponse.setHearingId(transcriptionEntity.getHearing().getId());
```

Comment:
Table "darts.transcription".hea_id is nullable - so probably need to check here to avoid NPE for migrated rows

Reply:
@richardbull done in a7f80468ce65394f97dbb75310a09cc4087e0381

I've also set the `"hearing_date"` from the transcription entity if there is no link to a hearing.